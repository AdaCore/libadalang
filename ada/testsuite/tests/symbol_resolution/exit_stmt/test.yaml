description:
    Symbol resolution: exit_stmt
driver: symbol_resolution
input_sources: [exit_stmt.adb]
