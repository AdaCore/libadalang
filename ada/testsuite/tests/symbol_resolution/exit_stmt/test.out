Analyzing exit_stmt.adb
#######################

Correct case
------------


Expr: <Id "Foo" 9:17-9:20>, references <SubpDecl ["Foo"] 2:4-2:31>, type is <EnumTypeDecl ["Boolean"] 3:3-3:32>

Incorrect case
--------------


Resolution failed for node <ExitStmt 13:7-13:20>

Done.
